### DISCUSSION TITLE
If it is possible to commit with PGP key sign?

### DISCUSSION THREAD

**Discussion by Jiaocz** [JamesIves/github-pages-deploy-action#1008] [Ideas]
Is it possible to add GPG secret key to action secrets and then sign commits using gpg key? Although it sounds unnecessary because it is just a github pages build

**Comment by JamesIves** [comment]
What would be your use case for such a feature? I'm curious to learn more about your situation and why you need such a thing. At first glance I feel like this might be a little counter intuitive but I could be wrong.

**Reply by Jiaocz** [reply]
In fact, it was a spur-of-the-moment idea, it won't do anything useful for but only a 'verified' icon in commit list on Github. So I think this feature will just for fun, or for those who just want to sign all commits in their repo. That's why I said it sounds unnecessary.
